fedora_commons/docker-entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
```

Comment:
Replacing the entrypoint *usually* makes it harder to upgrade the underlying tomcat image since we're not reusing (but rather copying) their entrypoint. Images like postgres and solr run all scripts stored in a specific directory, like /dockker-entrypoint.d . See http://www.camptocamp.com/en/actualite/flexible-docker-entrypoints-scripts/ for a Docker image design rationale and example.

However Tomcat does not use an entrypoint :). So fair play.